jobs/static/js/updating_user_job_posting/updating_job_posting.js

```diff
@@ -0,0 +1,51 @@
+// async function hideJob(job_obj_id) {
+//     setCookie("previously_selected_job_index", getCookie("currently_selected_job_index"))
+//     let data = {
+//         "id": job_obj_id,
+//         "hide": true
+//     }
+//     await updating_job_and_view(job_obj_id, data);
+// }
+async function toggle_applied(job_applied, job_obj_id, list_id, item_id) {
```

Comment:
move these functions out of here since they no longer use the UserJobPosting object